ios/FluentUI.Demo/FluentUI.Demo/Demos/DrawerDemoController.swift

```diff
@@ -143,6 +144,10 @@ class DrawerDemoController: DemoController {
         presentDrawer(sourceView: sender, presentationDirection: .up, contentView: containerForActionViews(), resizingBehavior: .dismissOrExpand)
     }
 
+    @objc private func showBottomDrawerWithExpandStateAndPassthroughBackgroundButtonTapped(sender: UIButton) {
+        navigationController?.pushViewController(PassThroughDrawerDemoController(), animated: true)
```

Comment:
In darkmode: can you make sure it looks good for passthrough drawercontroller? right now it is hard to tell between background and drawercontroller.

Reply:
Fixed!